datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -396,6 +396,13 @@ impl LogicalPlanBuilder {
         Self::scan_with_filters(table_name, table_source, projection, vec![])
     }
 
+    pub fn delim_get(table_index: usize, delim_types: &Vec<DataType>) -> Self {
```

Comment:
i think on real implementation we would need to update this function, for now let's keep it like this.
To be more specific in Datafusion there is no concept of table_index, maybe TableReference?